app_dart/test/request_handlers/reset_prod_task_test.dart

```diff
@@ -237,10 +261,6 @@ void main() {
       expect(task.attempts, equals(1));
     });
 
-    test('Fails if task does not exist', () async {
```

Comment:
Isn't this still true when the extra params are not passed?

Reply:
That use case is covered by "Re-schedule without all the parameters raises exception"